httpcore5/src/main/java/org/apache/hc/core5/http/ssl/TLS.java

```diff
@@ -94,4 +94,15 @@ public static ProtocolVersion parse(final String s) throws ParseException {
         return enabledProtocols.toArray(new String[0]);
     }
 
+    /**
+     * Check if a given protocol is considered secure and is enabled by default.
+     *
+     * @return {@code true} if the given protocol is secure and enabled, otherwise return {@code
+     * false}.
+     * @since 5.2
+     */
+    public static boolean isSecureAndEnabled(final String protocol) {
```

Comment:
@arturobernalg I think the name is still bad and misleading. Protocols are enabled by default because they are consdiered secure. You should change the name to `isSecureAndEnabledByDefault` or to `isSecure`.

Reply:
HI @ok2c 
I see. I change the name in favor of `isSecure`. 
TY

Reply:
> HI @ok2c I see. I change the name in favor of `isSecure`. TY

Yeah, had the same thought.